### ISSUE TITLE
Figure out predict on multiple series

### ISSUE THREAD

**Issue by sipemu** [DataZooDE/anofox-statistics#7]
When fitting a regression model on a training set with an ID column unique_id, then multiple scenarios may occur in the prediction step:

- There may be IDs in the test set with no equivalents in the training set or vice versa -> An INNER JOIN approach could solve this.
